nginx/nginx.conf

```diff
@@ -7,18 +7,35 @@ server {
         # Handle CORS headers
         proxy_set_header Host $host;
         proxy_set_header X-Real-IP $remote_addr;
-        add_header 'Access-Control-Allow-Origin' '*' always;
-        add_header 'Access-Control-Allow-Methods' 'GET, POST, PUT, DELETE, OPTIONS' always;
-        add_header 'Access-Control-Allow-Headers' 'Origin, X-Requested-With, Content-Type, Accept, Authorization' always;
-
-        # Allow OPTIONS method
-        if ($request_method = 'OPTIONS') {
-            add_header 'Access-Control-Allow-Origin' '*';
-            add_header 'Access-Control-Allow-Methods' 'GET, POST, PUT, DELETE, OPTIONS';
-            add_header 'Access-Control-Allow-Headers' 'Origin, X-Requested-With, Content-Type, Accept, Authorization';
-            add_header 'Content-Length' '0';
-            add_header 'Content-Type' 'text/plain charset=UTF-8';
-            return 204;
-        }
+        include 'snippets/cors_configs.conf'
```

Comment:
Please add ; at the end of 'include' lines

Reply:
Also it looks path should be adjusted:
`include '../snippets/cors_configs.conf';`